src/main/java/org/example/filetypes/CsvFile.java

```diff
@@ -0,0 +1,48 @@
+package org.example.filetypes;
+
+import com.opencsv.CSVWriter;
+import com.opencsv.bean.*;
+import org.example.fileData.FileData;
+import java.io.*;
```

Comment:
Wiem, że to InteliJ zmienia sam domyślnie, ale najlepiej jest importować tylko konkretne i wymagane klasy. Rano wam pokażę, jak w intelij zmienić aby te importy się tak nie zawijały